### ISSUE TITLE
Firefox 94 release causing CI test failures

### ISSUE THREAD

**Issue by nolanlawson** [salesforce/lwc#2565]
### Description

Since Firefox 94 was released, we're now seeing [CI test failures](https://app.circleci.com/pipelines/github/salesforce/lwc/4936/workflows/bca90dd5-f626-4b69-8b34-872bc53d759a/jobs/39871):

```
Firefox 94.0 (Windows 7) accessibility LightningElement.focus should not move focus if an internal element is already focused (delegatesFocus=true) FAILED
	Expected <input class="first"> to be <input class="second">. Tip: To check for deep equality, use .toEqual() instead of .toBe().
	<Jasmine>
	runFocusTests/<@accessibility/LightningElement.focus/test/accessibility/LightningElement.focus/index.spec.js:92:42 <- accessibility/LightningElement.focus/index.spec.js:168:46
	<Jasmine>
```

I'm assuming this is because they shipped [`delegatesFocus`](https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/delegatesFocus) in v94?

**Comment by nolanlawson** [comment]
Related: https://github.com/salesforce/lwc/issues/985

**Comment by nolanlawson** [comment]
Root cause: https://github.com/salesforce/lwc/issues/2566
